codeflash/cli_cmds/cmd_init.py

```diff
@@ -1373,7 +1356,7 @@ def ask_for_telemetry() -> bool:
     from rich.prompt import Confirm
 
     return Confirm.ask(
-        "⚡️ Would you like to enable telemetry to help us improve the Codeflash experience?",
+        "⚡️ Help us improve Codeflash by sharing anonymous usage data (e.g., commands run, errors encountered)?",
```

Comment:
we don't even get this data right now. only get posthog events. what do we get from sentry?